src/commands/cmd_context.h

```diff
@@ -7,29 +7,30 @@
 #pragma once
 
 #include "../redismodule.h"
+#include "../graph/graphcontext.h"
 #include "cypher-parser.h"
 
 /* Query context, used for concurent query processing. */
 typedef struct {
-	RedisModuleCtx *ctx;                  // Redis module context.
-	RedisModuleBlockedClient *bc;         // Blocked client.
-	char *graphName;                      // Graph ID.
-	char *query;                          // Query string.
-	RedisModuleString **argv;             // Arguments.
-	int argc;                             // Argument count.
-	bool replicated_command;              // Whether this instance was spawned by a replication command.
+	int argc;                       // Argument count.
+	char *query;                    // Query string.
+	RedisModuleCtx *ctx;            // Redis module context.
+	bool replicated_command;        // Whether this instance was spawned by a replication command.
+	GraphContext *graph_ctx;        // Graph context.
+	RedisModuleString **argv;       // Arguments.
+	RedisModuleBlockedClient *bc;   // Blocked client.
```

Comment:
Since `argc` and `replicated_command` are aligned, this order wastes 8 bytes. In general, I don't think this aesthetic preference is the most beneficial!